x/twap/logic.go

```diff
@@ -53,7 +53,12 @@ func getSpotPrices(ctx sdk.Context, k types.AmmInterface, poolId uint64, denom0,
 			sp1 = sdk.ZeroDec()
 		}
 	}
-	// if sp0.GT(gammtypes.MaxSpotPrice)
+	if sp0.GT(types.MaxSpotPrice) {
```

Comment:
this is just a safety check since `CalculateSpotPrice` shouldnt return values over types.MaxSpotPrice right?

Reply:
yup! In case a different spot price backend gets put in